recipe/meta.yaml

```diff
@@ -14,21 +14,32 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
+  skip: true  # [s390x]
```

Comment:
Why we skip on s390x?  It is a dependency, I guess.  So please give here a comment